### ISSUE TITLE
Is this project depreciated now that a official Prettier plugin for Tailwind CSS is available?

### ISSUE THREAD

**Issue by theDude823** [heybourn/headwind#185]

**Comment by darylknight** [comment]
No. Tailwind's first party plugin doesn't work with Prettier, so there's still a place for this.

**Comment by hacknug** [comment]
> No. Tailwind's first party plugin doesn't work with Prettier, so there's still a place for this.

It does work with Prettier because it is a plugin for it.

> Is this project deprecated now that a official Prettier plugin for Tailwind CSS is available?

I'd say it was abandoned by it's creator and probably not actively maintained anymore. As I mentioned on another issue, I personally started using [`eslint-plugin-tailwindcss`](https://github.com/francoismassart/eslint-plugin-tailwindcss) instead od headwind since it is activelly maintained and supports all of the latest features from core.

**Comment by ruridge** [comment]
Anyone else coming here after having issues with this plugin, I ended up replacing Headwind with ESLint+Prettier :
https://github.com/tailwindlabs/prettier-plugin-tailwindcss for class order formatting
https://github.com/francoismassart/eslint-plugin-tailwindcss for linting
